build.gradle

```diff
@@ -120,6 +120,10 @@ jar {
             '!org.testng.*',
             '!com.sun.*',
             '*'
+        instruction "Automatic-Module-Name", "org.testng"
+        attributes(
```

Comment:
It won't fix your issue but why do we need the duplication (instruction + attributes)?

Reply:
To be honest, I dont know Julien. When I tried adding just the
"instruction", I didnt see anything being added to the MANIFEST.MF and it
still showed some junk values.
The blog link that was shared in the issue recommends adding it up as an
instruction. But it didnt work for me (Perhaps due to a discrepancy with
the Gradle version maybe?)

That was why I went back to adding it as an attribute as well. After I
added it as an attribute, it worked but then cropped up this new problem of
IntelliJ not being able to resolve any of the classes within TestNG, when
consumed via a test project.

Reply:
Please remove the useless line before the merge.